scripts/fulfillURLManifest.py

```diff
@@ -4,24 +4,12 @@
 import logging
 from botocore.exceptions import ClientError
 from model import Link
-from managers import DBManager, s3
+from managers import DBManager
 
 s3_client = boto3.client("s3")
 
 bucketName = 'drb-files-qa'
 
-# class fulfillURLManifest():
-
-#     def __init__(self):
-
-#         # Connect to database
-#         self.generateEngine()
-#         self.createSession()
-
-#         # S3 Configuration
-#         self.s3Bucket = bucketName
-#         self.createS3Client()
-
```

Comment:
I changed my mind on using the putObjectInBucket method and chose to use the S3 client method put_object instead since these are QA manifests we're focusing on now so the extra error handling isn't needed for now and when the fulfill endpoints are integrated into future LA manifests in prod they will automatically use the putObjectInBucket method that's already in place.